client/src/app/core/ui-services/linenumbering.service.ts

```diff
@@ -377,27 +377,19 @@ export class LinenumberingService {
      * @returns {LineNumberRange}
      */
     public getLineNumberRange(html: string): LineNumberRange {
-        const cacheKey = this.djb2hash(html);
-        let range = this.lineNumberCache.get(cacheKey);
-        if (!range) {
-            const fragment = this.htmlToFragment(html);
-            range = {
-                from: null,
-                to: null
-            };
-            const lineNumbers = fragment.querySelectorAll('.os-line-number');
-            for (let i = 0; i < lineNumbers.length; i++) {
-                const node = lineNumbers.item(i);
-                const number = parseInt(node.getAttribute('data-line-number'), 10);
-                if (range.from === null || number < range.from) {
-                    range.from = number;
-                }
-                if (range.to === null || number + 1 > range.to) {
-                    range.to = number + 1;
-                }
+        const range = {
+            from: null,
+            to: null
+        };
+
+        let array1;
```

Comment:
There might be a better name than `array1` ;)

Reply:
It's kind of hard to argue with that... :-)
(Fixed in the new version of this PR)